lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,6 +27,24 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instance(instance_name)
```

Comment:
Ouch. I get it now. You have a `InstanceGroup` object and you are adding `Instance`'s to that object.
So GCE does abstract the collections into its own resource (`InstanceGroup`). I did not know that.

👍 then.